core/domain/stats_jobs_one_off.py

```diff
@@ -1373,3 +1373,52 @@ def reduce(key, items):
             yield (key, len(items))
         else:
             yield (key, items)
+
+
+class ExplorationMissingStatsAudit(jobs.BaseMapReduceOneOffJobManager):
+    """A one-off job for finding explorations that are missing stats models.
+
+    If this job outputs any explorations, then the following two jobs need to be
+    run in succession:
+        - RegenerateMissingV1StatsModelsOneOffJob
```

Comment:
Sorry, not clear which job you're talking about. Do you mean the one introduced by this PR? (If so: no). Or the jobs listed in this docstring? (If so: yes).

Reply:
I've made this clearer in the comment, WDYT?

Reply:
Actually, I don't want to tie the two sets of runs together. We should use this strictly to gather information, and should decide whether to run the regenerate stats in a separate discussion.

Because of this, I'm resolving this thread as "obsolete".